src/Query/Manipulation/HasFields.php

```diff
@@ -0,0 +1,52 @@
+<?php
+declare(strict_types = 1);
+
+namespace TBolier\RethinkQL\Query\Manipulation;
+
+use TBolier\RethinkQL\Query\AbstractQuery;
+use TBolier\RethinkQL\Query\Aggregation\AggregationTrait;
+use TBolier\RethinkQL\Query\Operation\OperationTrait;
+use TBolier\RethinkQL\Query\QueryInterface;
+use TBolier\RethinkQL\Query\Transformation\TransformationTrait;
+use TBolier\RethinkQL\RethinkInterface;
+use TBolier\RethinkQL\Types\Term\TermType;
+
+class HasFields extends AbstractQuery
```

Comment:
Can you rename the class to `HasFieldsLogic` to keep the naming similar to the existing `GetFieldLogic`?